TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/ManualMerge/CheckAnswers.cshtml

```diff
@@ -1,8 +1,8 @@
-@page "/persons/{personId}/merge/select-details-to-merge"
-@model TeachingRecordSystem.SupportUi.Pages.Persons.Merge.SelectDetailsToMergeModel
+@page "/persons/{personId}/manual-merge/check-answers"
```

Comment:
What's the reason for rename from 'merge' to 'manual merge'?

Reply:
Just that `/merge/merge/` seemed like a wierd path, and that `manual-merge` felt more explicit about the difference between this and the other merge journeys